MekHQ/src/mekhq/campaign/personnel/RetirementDefectionTracker.java

```diff
@@ -188,7 +188,8 @@ public static long getShareValue(Campaign campaign) {
     	for (Person p : campaign.getPersonnel()) {
     		if (!p.isActive() || p.isDependent()
     				|| p.isPrisoner() || p.isBondsman()
-    				|| p.isDeployed()) {
+    				|| p.isDeployed() ||
+					p.isFounder() && campaign.getCampaignOptions().getFoundersNeverRetire()) {
```

Comment:
Hate to nitpick, but let's make the || formatting consistent here.

Reply:
No worries, I would have nitpicked on it too if I saw it 👍